config/index.js

```diff
@@ -28,7 +28,8 @@ const config = {
       host: process.env.DB_HOST,
       port: process.env.DB_PORT,
       username: process.env.DB_USERNAME,
-      password: process.env.DB_PASSWORD
+      password: process.env.DB_PASSWORD,
+      name: process.env.DB_DATABASE
```

Comment:
the database name is set in the environment files is not necessary added to index file